lib/acl-checker.js

```diff
@@ -0,0 +1,230 @@
+'use strict'
+
+const async = require('async')
+const path = require('path')
+const url = require('url')
+
+const DEFAULT_ACL_SUFFIX = '.acl'
+
+class ACLChecker {
+  constructor (options = {}) {
+    this.debug = options.debug || console.log
+    this.fetch = options.fetch
+    this.strictOrigin = options.strictOrigin
+    this.suffix = options.suffix || DEFAULT_ACL_SUFFIX
+  }
```

Comment:
We could. But the whole point of going the RDFJS standard API route is so that there's only one match function, right? That's the one I've implemented in rdflib.js, and it's also being added to rdf-ext. Will that be sufficient, or do you still want to inject the match function?